docs/eigenda/operator-guides/upgrades/software-upgrades.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 1
+---
+
+# Software Upgrades
+
+Please monitor the following channels for updates to EigenDA Operator software:
+- [EigenLayer Discord](https://discord.gg/eigenlayer): #support-operators channel.
+- [EigenDA Operator Setup](https://github.com/Layr-Labs/eigenda-operator-setup) repository: [configure your watch settings](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository) for notifications of new releases.
+
+If you are running your node using docker compose, you can perform an upgrade by following the steps below:
```

Comment:
Do we want to provide a script to automate these steps? @shrimalmadhur 
Make upgrade low-effort will improve UX

Reply:
we could probably do a script, the only issue will be when there are more changes to `.env` like if we have some change/updates in some env then the script might not work, so that's why these steps here (which are standard docker steps) and explicitly here.